core/templates/server/static.tiddler.html.tid

```diff
@@ -20,10 +20,10 @@ title: $:/core/templates/server/static.tiddler.html
 </head>
 <body class="tc-body">
 <$transclude tiddler="$:/core/templates/server/static.sidebar.wikitext" mode="inline"/>
-<section class="tc-story-river" role="main">
-<div class="tc-tiddler-frame" role="article">
+<main class="tc-story-river">
```

Comment:
Again, the "tight" theme includes a reference to `section.tc-story-river`. The root issue is perhaps that this is a brittle technique for increasing specificity, but because it is used in the core we should expect third party themes to be using it too.